crypto/bn256/cloudflare/bn256_test.go

```diff
@@ -92,6 +92,64 @@ func TestTripartiteDiffieHellman(t *testing.T) {
 	}
 }
 
+func TestBinaryEAA(t *testing.T) {
+	for i := 0; i < 10000; i++ {
+		_, Ga, err := RandomG1(rand.Reader)
+		if err != nil {
+			t.Fatal(err)
+		}
+		tmpLittleFermat := &gfP{}
+		tmpLittleFermat.Invert(&Ga.p.x)
+
+		tmpBinaryEAA := &gfP{}
+		tmpBinaryEAA.EaaInvert(&Ga.p.x)
```

Comment:
Tests should also cover the identity-case, e.g. 
```
tmp2BinaryEEA = &Ga.p.x
tmp2BinaryEEA.EaaInvert(tmp2BinaryEEA)
```

Reply:
Added in the same test